frappe/email/doctype/notification/notification.js

```diff
@@ -161,6 +161,17 @@ frappe.ui.form.on("Notification", {
 	document_type: function (frm) {
 		frappe.notification.setup_fieldname_select(frm);
 	},
+	preview_document: (frm) => {
+		frappe.call({
```

Comment:
Maybe use `frm.call` here instead?

I think it's easier to read as you immediately see that a method of the doc gets called.

Reply:
I'll leave that to the merged, CI is green.